routes/create.py

```diff
@@ -19,5 +20,8 @@ async def create(request: web.Request):
     room = GameRoom(host, body['size'], body['selection'], body['medal'])
     asyncio.create_task(room.initialize_maplist())
     server.rooms.append(room)
-
-    return web.Response(text=room.code)
+    
+    return web.Response(text=dumps({
+        'room_code': room.code, 
+        'client_secret': host.secret
```

Comment:
I think it might be worth doing the secret exchange in the TCP opening handshake instead of over the HTTP API. That's where identification is handled, and supposedly authentification should be there too.